joplin/templates/wagtailadmin/base.html

```diff
@@ -9,7 +9,10 @@
 
     <div class="header">
         <div>
-            <img class="wagtail-logo wagtail-logo__full" src="{% static 'images/logo.svg' %}" alt="Welcome to Pearl" width="50" />
+            <img class="wagtail-logo wagtail-logo__full"
+              src="{% static 'images/coa_seal_transparent_white.png' %}"
+              alt="Welcome to Wagtail CMS" width="60"
```

Comment:
do we want it to say Wagtail? also I kinda miss the chicken but I guess we can be official

Reply:
I'm not sure. I was between that and Joplin CMS. But if there is a content editor or contract dev in the future, they could at least Google: "Wagtail CMS" and find their way to some docs and tutorials.

I'm kinda against the use of the love chicken outside our team. It's internal branding for our office which make us hypocritical whenever we tell other departments their brand doesn't matter. If this is a CMS for the whole city, we should use the City branding.

Reply:
I completely agree on the branding. Doesn't make me not miss the chicken though.

Reply:
I agree too, but the City Seal is too small to be legible, so also isn't a great choice. Maybe we can get permission to use a variation of this, which is far more web friendly?
![image](https://user-images.githubusercontent.com/3790067/43960217-f5bb563c-9c76-11e8-8251-63d3096878fa.png)

Reply:
I never liked the love chicken, so I'm not arguing with you on that point. However, I disagree that it doesn't matter if it's not legible. We're losing an opportunity to show them clean, well-designed internal tools. Internal audiences deserve good design too. We just don't have anything better at the moment.